AppAuth-JS/built/storage.d.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork